### DISCUSSION TITLE
Allow to collapse row groups by default

### DISCUSSION THREAD

**Discussion by RixzZ** [filamentphp/filament#10717] [Ideas]
The grouping rows feature that tables has is awesome. It allows to use tables to only display data in an organized way.

If you use tables to display grouped data, where the user might be only interested on seeing one of the groups, allowing to collapse the rest is a must.

What I'm missing is the possibility to collapse by default. That way the user, based on the group names, can expand only the one they are interested in.

**Comment by parisl69** [comment]
Has anyone found a way to collapse the groups by default?

**Comment by FoundCap** [comment]
+1 for this being added as an option in next version.

**Comment by synsyst** [comment]
Would like this feature as well  - as well as a way for the use to collapse-uncollapse all groups in table

**Comment by wbzy05** [comment]
also need this

**Comment by rozenlicht** [comment]
I'd also appreciate this as well!

**Comment by tiendinh595** [comment]
+1 for next version

**Comment by Worlonmight** [comment]
+1

**Comment by yannisc** [comment]
+1

**Comment by cataldo2014** [comment]
+1

**Comment by roncarino** [comment]
+1

**Comment by GeminiDev1** [comment]
This has been added as a feature as part of 4.1.0 :)

https://filamentphp.com/docs/4.x/tables/grouping#collapsing-groups-by-default
